src/python/pants/backend/jvm/tasks/jvm_compile/rsc/rsc_compile.py

```diff
@@ -454,10 +454,19 @@ def make_zinc_job(target, input_product_key, output_products, dep_keys):
         # Otherwise, fail it.
         on_success=ivts.update,
         on_failure=ivts.force_invalidate)
+        
+    workflow = rsc_compile_context.workflow
+
+    # Replica of JvmCompile's _record_target_stats logic
```

Comment:
you can remove this comment, in fact in a subsequent change you could just make `record_target_stats` public and make it take args and then use it here, since RscCompile inherits from JvmCompile. 

but this is good as is, and not a blocking change.